jwtauth_test.go

```diff
@@ -279,6 +279,23 @@ func TestMore(t *testing.T) {
 	}
 }
 
+func TestEncodeClaims(t *testing.T) {
+	claims := map[string]interface{}{
+		"key1": "val1",
+		"key2": 2,
+		"key3": time.Now(),
+		"key4": []string{"1", "2"},
+	}
+	claims[jwt.JwtIDKey] = 1
+	if _, _, err := TokenAuthHS256.Encode(claims); err == nil {
+		t.Fatal("encoding invalid claims succeeded")
+	}
```

Comment:
This now fails correctly per RFC-7519:

```
The "jti" value is a case-
   sensitive string.  Use of this claim is OPTIONAL.
```

https://datatracker.ietf.org/doc/html/rfc7519#page-10